usaon_vta_survey/templates/response/observing_systems.html

```diff
@@ -19,10 +19,12 @@ <h3>Observing systems</h3>
     {% endfor %}
   </table>
 
-  <h3>Add observing system</h3>
-  <form method="POST">
-    {{render_form(form, button_map={'submit_button': 'primary'}) }}
 
-  </form>
+  {% if not current_user.role_id=='analyst' %}
+    <h3>Add observing system</h3>
+    <form method="POST">
+      {{render_form(form, button_map={'submit_button': 'primary'}) }}
 
```

Comment:
The other response objects handle the "You do not have permissions" case. Let's do that here for consistency. Can we extract a macro so we don't have to keep them in sync?

Reply:
@MattF-NSIDC in the other views there is the + button in the table to add relationship. Here that does not exist so just creating a new system isn't available for view so it cannot be done.

Reply:
I'm not certain I follow this!

If the user has permissions, we're showing them the form titled `Add observing system`. It looks like we don't show anything if they don't have permissions, right?